tests/test_pic_dir.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import shutil
+import os
+import picturae_csv_create as pcc
+from tests.testing_tools import TestingTools
+from datetime import date, timedelta
+
+os.chdir("./image_client")
+
+class DirectoryTests(unittest.TestCase, TestingTools):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.date_string = self.test_date()
+
+
+    """WorkingDirectoryTests: a series of unit tests to verify
+        correct working directory, subdirectories."""
+    def setUp(self):
+        """setUP: unittest setup function creates empty csvs,
+                  and folders for given test path"""
+        # initializing
+        self.CsvCreatePicturae = pcc.CsvCreatePicturae(date_string=self.date_string, istesting=True)
+
+        if self._testMethodName == "test_missing_folder_raise_error":
+            pass
+        else:
+            # create test directories
+
+            expected_folder_path = 'picturae_csv/' + str(self.date_string) + '/picturae_folder(' + \
```

Comment:
This is the kind of thing (folder name) that belongs in a config file. Unless it's just for testing, and it's configuratble in the main code?

Reply:
I think I actually have this folder path format hard coded into the function in picturae_create_csv, I should change that there too , put it in a config file. And change both.